sdk/core/azure-core/azure/core/_pipeline_client.py

```diff
@@ -204,9 +193,9 @@ def send_request(self, request, **kwargs):
         :return: The response of your network call. Does not do error handling on your response.
         :rtype: ~azure.core.rest.HttpResponse
         # """
-        rest_request, request_to_run = _prepare_request(request)
+        rest_request = hasattr(request, "content")
         return_pipeline_response = kwargs.pop("_return_pipeline_response", False)
```

Comment:
Do we still need it?

Reply:
Or maybe you want to deal with it in another PR?

Reply:
we're still going to keep `_return_pipeline_response`, autorest needs it. Will fix the awaitble context manager issue here though in another PR